packages/@aws-cdk/aws-iot-actions/test/sns/sns-topic-action.test.ts

```diff
@@ -0,0 +1,104 @@
+import { Match, Template } from '@aws-cdk/assertions';
+import * as iam from '@aws-cdk/aws-iam';
+import * as iot from '@aws-cdk/aws-iot';
+import * as sns from '@aws-cdk/aws-sns';
+import * as cdk from '@aws-cdk/core';
+import * as actions from '../../lib';
+
+const SNS_TOPIC_ARN = 'arn:aws:sns::123456789012:test-topic';
+const RULE_ROLE_ID = 'MyTopicRuleTopicRuleActionRoleCE2D05DA';
```

Comment:
I would probably inline this variable (it's exact value depends on the specifics of the test it's used in, for example, it won't be generated in the explicit Role test).